js/js_test.go

```diff
@@ -0,0 +1,12 @@
+package js
+
+import "testing"
+
+func BenchmarkNewOttoVM(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		_, _ = newOttoVM(nil)
+		for k := range ottoCache.vms {
+			delete(ottoCache.vms, k)
+		}
```

Comment:
Should clearing that `ottoCache.vms` be part of what is measured?
You might want to start/stop the clock around `newOttoVM()` only.